shader/flatten.wgsl

```diff
@@ -85,6 +173,11 @@ fn read_i16_point(ix: u32) -> vec2<f32> {
     return vec2(x, y);
 }
 
+struct Transform {
+    matrx: vec4<f32>,
```

Comment:
It's a keyword in WGSL. And I don't know yet the answer to that question, it depends on how the math works out. An affine could be stored as a mat3x2f, but I chose not to do that so that the transform could be written without having to multiply by 1.

I'll go with `mat` as I do like conciseness.